### PR TITLE
Add --prefix-lines-with option

### LINKED ISSUE #10: Add "--wrap" type option
Frequently you want to run the commands inside a docker container but you don't want to install tuttest inside the docker container. It would be good if `tuttest` had a `--wrap` or `--prefix` type option which would prefix / run the commands with the option.

For example;
```
tuttest --wrap "docker run --abc 1234" XYZ
```
Would prefix all the commands with `docker run --abc 1234`.

### PR DESCRIPTION
**Pull request by kamilrakoczy**
This PR adds ``--prefix-lines-with`` option.

Fixes #10 

Signed-off-by: Kamil Rakoczy <krakoczy@antmicro.com>

### CODE REVIEW

**Comment by mgielda** [comment]
@kgugala this will close #10 - could we get it merged?

File: bin/tuttest [hunk 1]
```diff
@@ -4,31 +4,49 @@ from tuttest import get_snippets
...
 
+    if args.prefix_lines_with:
+        prefixed_code = []
+        for snippet in code:
+            prefixed_snipped = args.prefix_lines_with + " \'"
+            for line in snippet.splitlines():
+                if len(line.strip()): # skip empty lines
+                    prefixed_snipped += line + ";"
+            prefixed_snipped += "\'"
+            prefixed_code.append(prefixed_snipped)
```
Comment (acomodi) [inline bin/tuttest]: Snipped -> snippet

File: README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ i.e., a Travis CI script. Here is a synopsis of a direct `tuttest` call:
     if you do not name a snippet, it will receive the `unnamedX` name, where
     `X` is the snippet number in the file (starting from 0). Examples of named
     snippets can be found below.
+  * `<prefix>` is a command, that snippet should be prefixed with.
```
Comment (acomodi) [inline README.md]:
```suggestion
  * `<prefix>` is a command that is added to the snippet as a prefix.
```

**Review by acomodi — CHANGES_REQUESTED** [review]
Comments in

File: bin/tuttest (hunk 1 above)
Comment (kamilrakoczy) [reply bin/tuttest]: fixed

File: README.md (hunk 2 above)
Comment (kamilrakoczy) [reply README.md]: fixed

**Comment by acomodi** [comment]
@kamilrakoczy I am thinking that it is worth adding a test for the `--prefix-lines-with` feature

**Review by acomodi — APPROVED** [review]
LGTM

**Pull request merged**